Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs

```diff
@@ -1777,10 +1786,33 @@ private void HideAllborders()
             ALMDefectsBorder.BorderBrush = null;
         }
         private void xRunsetSaveBtn_Click(object sender, RoutedEventArgs e)
-        {            
-            SaveRunSetConfig();           
+        {
+
+            if (mRunSetConfig.GingerRunners.Any(x => x.Status == eRunStatus.Stopped) && Reporter.ToUser(eUserMsgKey.SaveRunsetChangesWarn) == eUserMsgSelection.Yes)
+            {
+                ResetALMDefectsSuggestions();
```

Comment:
Move this code under  1 common reset function. And use the same in Save as well as Reset event